es6-shim.js

```diff
@@ -1594,6 +1604,10 @@
     Math.imul = MathShims.imul;
   }
 
+  if (Math.log2(536870912.0) !== 29) {
+    Math.log2 = MathShims.log2;
```

Comment:
@ljharb , assignment preservers enumerability (previously I thought it sets all flags to "true", but it preservers it, seems)